### PR TITLE
use ubi8 go-toolset image for docker build and use sprintf for string build

### PR DESCRIPTION
**Pull request by aizuddin85**
### What does this PR do?
* Docker build now use dockerhub golang 1.14 
* use sprintf to build string

### How should this be tested?
As per README.md

### Is there a relevant Issue open for this?
Provide a link to any open issues that describe the problem you are solving.
N/A

### Other Relevant info, PRs, etc.
N/A

### People to notify
cc: @oybed

### CODE REVIEW

File: alertmanager-notifier-gateway/alertmanager-line-gateway/README.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ Line Notification Gateway for OpenShift Prometheus Alertmanager.
 
 ## Installation ##
 
+# Prebuilt image  
+```
+https://quay.io/repository/redhat/alertmanager-line-gateway?tab=tags
```
Comment (garethahealy) [inline alertmanager-notifier-gateway/alertmanager-line-gateway/README.md]: who owns this repo? all the CoP images live under redhat-cop.

**Comment by aizuddin85** [comment]
I dont have access to redhat-cop org in quay.io. Can someone add me for access or point me on how to obtain an access. For now I will remove this line as this is non CoP repository.

**Comment by aizuddin85** [comment]
I dont have access to redhat-cop org in quay.io. Can someone add me for access or point me on how to obtain an access. For now I will remove this line as this is non CoP repository.

**Review by oybed — APPROVED** [review]
LGTM

**Comment by oybed** [comment]
@aizuddin85 for any additional access requests, please submit an issue for the `org` repo:
https://github.com/redhat-cop/org/issues/new/choose

**Pull request merged**